card/mod/02_basic_types/set/type/pointer.rb

```diff
@@ -218,6 +218,14 @@ def default_item_view
   end
 end
 
+format :rss do
+  def raw_feed_items args
```

Comment:
Unused method argument - `args`. If it's necessary, use `_` or `_args` as an argument name to indicate that it won't be used. You can also write as `raw_feed_items(*)` if you want the method to accept any arguments but don't care about them.<br>Use def with parentheses when there are parameters.